vendor/istio.io/api/networking/v1alpha3/service_entry.proto

```diff
@@ -59,7 +59,7 @@ option go_package = "istio.io/api/networking/v1alpha3";
 //     metadata:
 //       name: mtls-mongocluster
 //     spec:
-//       host: mymongodb.somedomain
+//       name: mymongodb.somedomain
```

Comment:
Far too late to go back to the old name.

Reply:
Fixed in istio.io/api PR. Will get picked up when I fixup the vendoring in this PR (after the API PR is merged)